spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -143,16 +146,18 @@ def get_parser():
 class AnalyzeLesion:
     def __init__(self, fname_mask, fname_sc, fname_ref, path_template, path_ofolder, perslice, verbose):
         self.fname_mask = fname_mask
-        self.midsagittal_sc_slice = None
-        self.midsagittal_sc_slice_rpi = None
         self.fname_sc = fname_sc
         self.fname_ref = fname_ref
         self.path_template = path_template
         self.path_ofolder = path_ofolder
         self.verbose = verbose
         self.wrk_dir = os.getcwd()
         # NOTE: the tissue bridges are NOT included in self.measure_keys because we do not want to average them
-        self.measure_keys = ['volume [mm3]', 'length [mm]', 'max_equivalent_diameter [mm]', 'max_axial_damage_ratio []']
+        self.measure_keys = ['volume [mm3]', 'length [mm]', 'width [mm]',
+                             'max_equivalent_diameter [mm]', 'max_axial_damage_ratio []']
+        self.interpolated_midsagittal_slice = None      # target float sagittal slice number used for the interpolation. This number is based on the spinal cord center of mass.
+        self.interpolation_slices = None                # sagittal slices used for the interpolation
+        self.interpolation_factor = None                # factor used for the interpolation of sagittal slices
```

Comment:
This is just a nitpick, but I like making changes that allow the PR diff to be easy to read.

Here, instead of adding the new attributes to the end of the list, we could put them in the same location the old attributes used to be. That way, it is clear that we went from `self.midsagiital_sc_slice` to `self.interpolated_midsagittal_slice`.

Reply:
fair point. thank you. reordered in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4702/commits/943c83a5aca2203b431741e6f7143022b4f4e612